src/excmds.ts

```diff
@@ -2900,20 +2900,16 @@ export async function hint(option?: string, selectors?: string, ...rest: string[
         let link = await hinting.pipe_elements(DOM.anchors())
         anchorUrl.hash = link.id || link.name
         run_exstr("yank " + anchorUrl.href)
-    } else if (option === "-c") DOM.simulateClick(await hinting.pipe(selectors))
+    } else if (option === "-c") DOM.simulateClick(await hinting.pipe(selectors)[0])
     // Deprecated: hint exstr
     else if (option === "-W") run_exstr(selectors + " " + rest.join(" ") + " " + (await hinting.pipe(DOM.HINTTAGS_selectors)))
     else if (option === "-pipe") return (await hinting.pipe(selectors))[rest.join(" ")]
     else if (option === "-br") {
         let lasthref = ""
         let starttime = window.performance.now()
```

Comment:
Delete me and the line above

Reply:
Uuh you want both line 2903 and 2902 gone? or just 2903?

Reply:
On https://github.com/cmcaine/tridactyl/pull/860/files it looks like bovine3dom is talking about 2908 and 2909.